config.go

```diff
@@ -389,6 +389,7 @@ func loadConfig() (*config, error) {
 			Dir:     defaultLitecoindDir,
 			RPCHost: defaultRPCHost,
 		},
+		UnsafeDisconnect:   true,
```

Comment:
Can't the field be completely removed now?

Reply:
that would break existing conf files or CLI commands, it will be fully removed in a later PR after more people have had enough time to adjust for the deprecation

Reply:
Oh yes, ofc, it can still be flagged